tests/PHPUnit/Integration/SegmentTest.php

```diff
@@ -426,6 +426,92 @@ public function test_getSelectQuery_whenJoinLogLinkVisitActionOnActionOnVisit_Wi
         $this->assertEquals($this->removeExtraWhiteSpaces($expected), $this->removeExtraWhiteSpaces($query));
     }
 
+    public function test_getSelectQuery_whenJoiningManyCustomTablesItShouldKeepTheOrderAsDefined()
+    {
+        $actionType = 3;
+        $idSite = 1;
+        $select = 'log_link_visit_action.custom_dimension_1,
```

Comment:
I had a similar case where it resulted in errors when executing the query depending on the PHP version